pytest_retry/retry_plugin.py

```diff
@@ -209,7 +211,7 @@ def pytest_runtest_makereport(
         t_call = pytest.CallInfo.from_call(
             lambda: hook.pytest_runtest_teardown(
                 item=item,
-                nextitem=pytest.Item.from_parent(item.session, name="fakeboi"),
+                nextitem=pytest.Module.from_parent(parent=item.session, path=pathlib.Path(os.devnull)),
```

Comment:
I set the line length to 99 for this repo and this goes over slightly. Would just need to be reformatted before merging, but unless you have any objections I think I'm going to close this and merge #35 instead.

Reply:
Would also want to update the comment on 209 to indicate Module instead of Item

Reply:
I have no objections. Thank you for moving this forward.

I actually chose to keep the comment, because what is important is that it is an `Item`, not necessarily a `Module`, which I chose over `Class` without knowing any of the trade-offs.